packages/gatsby-admin/src/components/navbar.tsx

```diff
@@ -33,7 +33,7 @@ const Navbar: React.FC<{}> = () => {
             sx={{
               width: `1px`,
               height: `16px`,
-              backgroundColor: `grey.40`,
+              backgroundColor: `whiteFade.50`,
```

Comment:
Any reason this is a specific color?

Reply:
Yeah, we don't have a color for this kind of layout line on a purple background! whiteFade.50 _should_ work on most colors, or at least more than `grey.40`. Great question!